common/src/androidTest/java/com/microsoft/identity/common/Util.java

```diff
@@ -23,11 +23,19 @@
 
 package com.microsoft.identity.common;
 
+import android.content.pm.PackageInfo;
+import android.content.pm.Signature;
+import android.content.pm.SigningInfo;
+import android.os.Build;
```

Comment:
```suggestion
import android.os.Build;

import androidx.annotation.Nullable;
```

Reply:
As a question, should we be using @CheckForNull?

Reply:
Hmm.... I'm not familiar with it! Does the Android Java implementation have this?

![image](https://user-images.githubusercontent.com/990063/110049612-05c56e00-7d07-11eb-9615-e4c13fd5d0dc.png)

Reply:
No, but Lombok does. :-)